src/main/java/co/com/bancolombia/factory/upgrades/UpgradeMainGradle.java

```diff
@@ -0,0 +1,40 @@
+package co.com.bancolombia.factory.upgrades;
+
+import static co.com.bancolombia.Constants.MainFiles.MAIN_GRADLE;
+
+import co.com.bancolombia.factory.ModuleBuilder;
+import java.util.concurrent.atomic.AtomicBoolean;
+import lombok.SneakyThrows;
+
+public class UpgradeMainGradle implements UpgradeAction {
+
+  @Override
+  public boolean up(ModuleBuilder builder) {
```

Comment:
Are you using this default implementation? If not, you can delete it and make this class abstract.

Reply:
you are right, solved